README.md

```diff
@@ -43,12 +45,29 @@ algeria = Everypolitician.country('Algeria')
 national_assembly = algeria.legislature('Majlis')
 ```
 
-If you want to point at a different `countries.json`, e.g. a local path or a different url, you can set it like this:
+By default, the gem connects to EveryPolitician's data on GitHub over HTTPS and
+returns the most recent data. Specifically it uses the current index file,
+called `countries.json`, which itself contains links to specific versions of
+data files.
+
+If you want to point at a different `countries.json`, such as one you've
+already downloaded, you can override this default behavour by setting it like
+this:
```

Comment:
Is this deliberately hiding the approach of pointing at Github, but not at `master`?